test/functional/collection.test.js

```diff
@@ -799,7 +799,7 @@ describe('Collection', function() {
 
           db.listCollections().toArray((err, documents) => {
             expect(err).to.not.exist;
-            expect(documents.length > 1).to.be.true;
+            expect(documents.length >= 1).to.be.true;
```

Comment:
what changed here?

Reply:
This test fails if you run it in isolation, it depends upon other tests to create multiple collections when it only creates one itself.